tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1238,7 +1238,12 @@ handle_build() {
 				kernel_headers_dir=$(get_kernel_headers_dir "${build_target}")
 
 				pushd "${kernel_headers_dir}"
-				find . -type f -name "*.${KERNEL_HEADERS_PKG_TYPE}" -exec tar cvfJ "${kernel_headers_final_tarball_path}" {} +
+				find . -type f -name "*.${KERNEL_HEADERS_PKG_TYPE}" -exec tar rvf kernel-headers.tar {} +
+				mv kata-linux-*/certs/signing_key.pem .
+				mv kata-linux-*/certs/signing_key.x509 .
```

Comment:
Most likely I missed it, but where do you use the .x509 file again?

Reply:
This is used implicilty bye the modules_install kernel infrastructure you need the pair to sign modules correctly even if you do not specify it explicitly it is used by the kernel Makefiles aka sign utility.